consensus/polybft/contractsapi/init_nexus.go

```diff
@@ -0,0 +1,30 @@
+package contractsapi
```

Comment:
Move this initialization file  somewhere in e2e test framework. its only part of the test